osidb/tests/test_flaw.py

```diff
@@ -914,3 +919,26 @@ def test_validate_embargoing_public_flaw(self):
             ]
             flaw.unembargo_dt = tzdatetime(2022, 1, 1)
             flaw.save()
+
+    @pytest.mark.parametrize(
+        "cwe,should_raise",
+        [
+            (None, False),
```

Comment:
We do not represent empty string based fields (CharField, TextField, etc.)  in OSIDB by `None` (NULL in db) since it creates a mess (you can read about it [here](https://docs.djangoproject.com/en/4.1/ref/models/fields/#django.db.models.Field.null)). Currently we do have a compatibility shim https://github.com/RedHatProductSecurity/osidb/blob/master/osidb/mixins.py#L54 which converts all `None` values in string based fields to an empty string because Bugzilla and SFM2 stores  `None` values in db. This will go away one day and thus this test will then fail since it would violate the database integrity. I would drop the test with None.

Reply:
understood, I have updated, thank you!